conda/recipe/0002-remove-futures.patch

```diff
@@ -0,0 +1,25 @@
+From 5cfd191fc377979755865de3cd15f13321204cef Mon Sep 17 00:00:00 2001
+From: "Andres A. Lugo-Reyes" <aalugore@us.ibm.com>
+Date: Tue, 23 Jul 2019 15:27:37 +0000
+Subject: [PATCH] Remove futures install dependency
+
+---
+ dali/python/setup.py.in | 3 ---
+ 1 file changed, 3 deletions(-)
+
+diff --git a/dali/python/setup.py.in b/dali/python/setup.py.in
+index 13a0a86..99d2adf 100644
+--- a/dali/python/setup.py.in
++++ b/dali/python/setup.py.in
+@@ -34,8 +34,5 @@ setup(name='nvidia-dali@DALI_FLAVOR_MINUS@',
+               'rec2idx = rec2idx:main',
+               ],
+           },
+-      install_requires = [
```

Comment:
Do it breaks anything in your case/setup. Because for Python 2.7 is is needed.

Reply:
OK. I see how it works now. This setup is not real installation step, it is used only to prepare a conda package.